docs/1.18.x/setup/installation/java.md

```diff
@@ -48,12 +48,17 @@ If no errors have appeared, you have successfully installed the Java JDK!
     * [x] Installing Java
     * [ ] Installing An IDE
     * [ ] Downloading Forge
+    * [ ] Running your mod
 - [ ] Configuration
     * [ ] Buildscript
     * [ ] Gradlew Commands
     * [ ] Mod File
     * [ ] Main Class
-- [ ] Conclusion
+- [ ] Basic Content
```

Comment:
This should not be here, not part of the setup tutorial. Blocks and items is a seperate tutorial.